### DISCUSSION TITLE
cell size not consistent (beta)

### DISCUSSION THREAD

**Discussion by imed-ghomari** [RafaelGB/obsidian-db-folder#217] [General]
**Describe the bug**
When adding a select option to only some cells in compact mode, the cell size becomes different from the other cells, which makes things inconsistent.

**To Reproduce**
1. create database
2. change cell size to compact in database settings
3. create select column
4. add select option to some rows

**Expected behavior**
The cell size should be consistent even after editing.

**Screenshots**

https://user-images.githubusercontent.com/38974541/180305809-2304e8a3-67d9-4b97-8996-ddfdf89c600d.mov

**Desktop (please complete the following information):**
 - macOS 12.3.1
 - Obsidian 0.15.6
 - DB folder 2.0.0-beta.2

**Comment by RafaelGB** [comment]
This is the expected behaviour of the row. To grow in function of the biggest cell

**Reply by imed-ghomari** [reply]
For consistency, I think all the rows should have the same size. For example, if one row grows bigger because it contains a select option (like in the video) the other rows should match its size.

**Reply by RafaelGB** [reply]
For now the idea its to maintain this behaviour. Work just like excel or Notion tables, every row could have its own size in function of the content
